core/services/keystore/models.go

```diff
@@ -324,8 +342,9 @@ func (rawKeys rawKeyRing) keys() (*keyRing, error) {
 		keyRing.OCR[ocrKey.ID()] = ocrKey
 	}
 	for _, rawOCR2Key := range rawKeys.OCR2 {
-		ocr2Key := rawOCR2Key.Key()
-		keyRing.OCR2[ocr2Key.ID()] = ocr2Key
+		if ocr2Key := rawOCR2Key.Key(); ocr2Key != nil {
+			keyRing.OCR2[ocr2Key.ID()] = ocr2Key
+		}
```

Comment:
Do we want to report that we found a key that we couldn't bundle here?  This might be of value as we start adding more chains rapidly (and potentially in the future chains we aren't directly aware of - which means we'll have other methods for bundling - but we may still get a nil here if we make a mistake elsewhere.)